src/Tasks/Common/Resources/Strings.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why all the formatting changes o this file?

Reply:
VS started doing this at some point. I think we should just take it because I've reverted it myself a bunch of times. Would have preferred to do it separately, but let's get it over with.